src/pages/therapies/index.js

```diff
@@ -9,8 +9,8 @@ export const TherapiesPage = () => {
     <>
       <NavBar />
       <TherapiesCarrossel />
-        <Parceires />
-        <Footer />
-        </>
+      <Parceires />
```

Comment:
Fica para escopo de outra PR, mas aqui seguimos usando pt-BR misturado com inglês.

```suggestion
      <Partners />
```